androidtv/basetv/basetv.py

```diff
@@ -167,6 +167,14 @@ def _cmd_current_app(self):
         ):
             return constants.CMD_CURRENT_APP_GOOGLE_TV
 
+        # Is this an Askey STI6130 Device?
+        if (
+            self.DEVICE_ENUM == constants.DeviceEnum.ANDROIDTV
+            and "askey" in self.device_properties.get("manufacturer")
+            and "sti6130" in self.device_properties.get("product_id")
```

Comment:
```suggestion
            and "askey" in self.device_properties.get("manufacturer", "")
            and "sti6130" in self.device_properties.get("product_id", "")
```

Reply:
Thanks for reviewing! Committed your suggestion.